api/services/control/control.pb.go

```diff
@@ -9,14 +9,14 @@ import (
 	_ "github.com/gogo/protobuf/gogoproto"
 	proto "github.com/gogo/protobuf/proto"
 	github_com_gogo_protobuf_types "github.com/gogo/protobuf/types"
-	_ "github.com/golang/protobuf/ptypes/timestamp"
 	types "github.com/moby/buildkit/api/types"
 	pb "github.com/moby/buildkit/solver/pb"
 	github_com_moby_buildkit_util_entitlements "github.com/moby/buildkit/util/entitlements"
 	github_com_opencontainers_go_digest "github.com/opencontainers/go-digest"
 	grpc "google.golang.org/grpc"
 	codes "google.golang.org/grpc/codes"
 	status "google.golang.org/grpc/status"
+	_ "google.golang.org/protobuf/types/known/timestamppb"
```

Comment:
Updated protoc in a separate commit for visibility, and in case this change is problematic

Reply:
Looks like it's problematic indeed; 

```
#28 [buildctl 1/1] RUN --mount=target=. --mount=target=/root/.cache,type=cache   --mount=target=/go/pkg/mod,type=cache   --mount=source=/tmp/.ldflags,target=/tmp/.ldflags,from=buildkit-version   xx-go build -ldflags "$(cat /tmp/.ldflags)" -o /usr/bin/buildctl ./cmd/buildctl &&   xx-verify --static /usr/bin/buildctl
#28 sha256:43c1d64f11a2dca8edd62a6ac448d7355a481e1faa15d331e267849e86aba17e
#28 0.434 build google.golang.org/protobuf/types/known/timestamppb: cannot load google.golang.org/protobuf/types/known/timestamppb: open /src/vendor/google.golang.org/protobuf/types/known/timestamppb: no such file or directory
#28 ERROR: executor failed running [/bin/sh -c xx-go build -ldflags "$(cat /tmp/.ldflags)" -o /usr/bin/buildctl ./cmd/buildctl &&   xx-verify --static /usr/bin/buildctl]: exit code: 1
```

Reply:
Let me update to 3.11.4 instead (version used by google/protobuf); https://github.com/golang/protobuf/blob/v1.3.5/.travis.yml#L15